semantic-conventions/src/tests/semconv/model/test_correct_parse.py

```diff
@@ -727,6 +727,68 @@ def test_scope_attribute(self):
         }
         self.semantic_convention_check(list(semconv.models.values())[0], expected)
 
+    def test_reference_only_ref(self):
+        semconv = SemanticConventionSet(debug=False)
+        semconv.parse(self.load_file("yaml/http.yaml"))
+        self.assertEqual(len(semconv.models), 3)
+        semconv.parse(self.load_file("yaml/general.yaml"), True)
```

Comment:
```suggestion
        semconv.parse(self.load_file("yaml/general.yaml"), ref_only=True)
```

etc
(or see comment before for maybe using an API that does not rely on a boolean)